stubs/fpdf2/fpdf/graphics_state.pyi

```diff
@@ -32,25 +32,25 @@ class GraphicsStateMixin:
     @font_stretching.setter
     def font_stretching(self, v: float) -> None: ...
     @property
-    def char_spacing(self): ...
+    def char_spacing(self) -> float: ...
     @char_spacing.setter
-    def char_spacing(self, v) -> None: ...
+    def char_spacing(self, v: float) -> None: ...
     @property
-    def font_family(self): ...
+    def font_family(self) -> str: ...
     @font_family.setter
-    def font_family(self, v) -> None: ...
+    def font_family(self, v: str) -> None: ...
     @property
-    def font_size_pt(self): ...
+    def font_size_pt(self) -> float: ...
     @font_size_pt.setter
-    def font_size_pt(self, v) -> None: ...
+    def font_size_pt(self, v: float) -> None: ...
     @property
-    def font_size(self): ...
+    def font_size(self) -> float: ...
     @font_size.setter
-    def font_size(self, v) -> None: ...
+    def font_size(self, v: float) -> None: ...
     @property
-    def current_font(self): ...
+    def current_font(self) -> float: ...
     @current_font.setter
-    def current_font(self, v) -> None: ...
+    def current_font(self, v: float) -> None: ...
```

Comment:
`current_font` doesn't look like a `float` to me. It's set to a dictionary in the following places:

- https://github.com/PyFPDF/fpdf2/blob/0b40fca2cc119d76e08551c0f4e74e7131a95584/fpdf/fpdf.py#L323
- https://github.com/PyFPDF/fpdf2/blob/0b40fca2cc119d76e08551c0f4e74e7131a95584/fpdf/fpdf.py#L1894

And it's assumed to be a dictionary in the following places:

- https://github.com/PyFPDF/fpdf2/blob/0b40fca2cc119d76e08551c0f4e74e7131a95584/fpdf/fpdf.py#L386
- https://github.com/PyFPDF/fpdf2/blob/0b40fca2cc119d76e08551c0f4e74e7131a95584/fpdf/fpdf.py#L1896
- https://github.com/PyFPDF/fpdf2/blob/0b40fca2cc119d76e08551c0f4e74e7131a95584/fpdf/fpdf.py#L1913
- https://github.com/PyFPDF/fpdf2/blob/0b40fca2cc119d76e08551c0f4e74e7131a95584/fpdf/fpdf.py#L2318

I think `dict[str, Any]` would be a better type hint for `current_font`.

Reply:
you are right..i will do the changes.